ui/admin/app/components/form/onboarding/index.js

```diff
@@ -0,0 +1,32 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+export default class FormOnboardingComponent extends Component {
+  // =properties
+  @tracked targetAddress;
+  @tracked targetAddressInvalid = false; // change to method
+  @tracked targetPort;
+  @tracked targetPortInvalid = false; // change to method name
+
+  // =actions
+  @action
+  // change name to formSubmit
+  submit() {
+    // Check targetAddress and targetPort are valid before submit
+    this.targetAddressInvalid = this.requiredFieldInvalid(this.targetAddress);
+    this.targetPortInvalid = this.requiredFieldInvalid(this.targetPort);
```

Comment:
It might make more sense to just forgo the function and just do `!this.targetAddress` and `!this.targetPort` since it's not doing much

And a comment that this is necessary so that for the initial render it doesn't think the fields are invalid

Reply:
You are right. I will quickly change it, run manual testing and push this change. Thanks for the catch!

Reply:
@ZedLi Actually this is neccessary, this is the only moment where the set state for `targetAddressInvalid` and `targetPortInvalid`

Reply:
Sorry I missundertood the comment. Yes `requiredFieldInvalid` is not need at all, refactored to not use it.